webapp/lib/model.g.dart

```diff
@@ -349,6 +349,7 @@ class Tag {
   String text;
   TagType type;
   String shortcut;
+  bool filterable;
```

Comment:
Wouldn't hurt to add this to `model.yaml` as well... but not required

Reply:
Good point, done, thanks!